lib/challenge.rb

```diff
@@ -1,19 +1,6 @@
-MORSE_CODE = {
-  '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
-  '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
-  '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
-  '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
-  '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
-  '--..' => 'Z', '-----' => '0', '.----' => '1', '..---' => '2', '...--' => '3',
-  '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
-  '---..' => '8', '----.' => '9', '.-.-.-' => '.', '--..--' => ',',
-  '..--..' => '?', '.----.' => "'", '-.-.--' => '!', '-..-.' => '/',
-  '-.--.' => '(', '-.--.-' => ')', '.-...' => '&', '---...' => ':',
-  '-.-.-.' => ';', '-...-' => '=', '.-.-.' => '+', '-....-' => '-',
-  '..--.-' => '_', '.-..-.' => '"', '...-..-' => '$', '.--.-.' => '@',
-  '...---...' => 'SOS'
-}.freeze
-## DO NOT CHANGE ANYTHING ABOVE THIS LINE
+# frozen_string_literal: false
+
+require_relative '../lib/morse_code'
```

Comment:
This can be changed to `require_relative ./morse_code` since the file is in the same directory.